src/main/java/uk/gov/companieshouse/transactions/web/exception/ServiceException.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.companieshouse.transactions.web.exception;
+
+/**
+ * The class {@code ServiceException} is a form of {@code Exception}
+ * that should be used at the service layer to abstract lower level
+ * exceptions from being propagated up the call stack.
+ */
+public class ServiceException extends Exception {
```

Comment:
Is it worth adding a `ServiceException` that takes only a string? This is for those who want to throw a service layer exception without having a throwable cause.

Reply:
Done